theano/misc/tests/test_pycuda_theano_simple.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
I've moved it below any module-level docstring in all the files affected, and rebased again. PTAL.